src/main.cpp

```diff
@@ -0,0 +1,19 @@
+#include "zed_cpu.hpp"
+
+int main(int argc, char * argv[])
+{
+  ros::init(argc, argv, "zed_camera");
+
+  auto nh = std::make_shared<ros::NodeHandle>("~");
+  auto it = std::make_shared<image_transport::ImageTransport>(*nh);
+
+  ZedCameraNode zed_camera_node(nh, it);
```

Comment:
```suggestion
  zed_cpu::ZedCameraNode zed_camera_node(nh, it);
```
probably won't work without this.